formatting.css

```diff
@@ -0,0 +1,231 @@
+body, html{
+    margin: 0%;
+    padding: 0%;
+}
+.header-banner{
+    position: relative;
+    text-align: center;
+    background-color: darkgrey;
+    width: 100%;
+}
+.header-banner .title-alligner{
+    position: absolute;
+    top: 50%;
+    left: 25%;
+    transform: translate(-50%, -50%);
+    width: 45%;
+}
+.header-banner .cart-alligner{
+    position: absolute;
+    top: 25%;
+    left: 85%;
+    transform: translate(-50%, -50%);
+    color: white;
+    font-size: 2em;
+}
+.cart-alligner input {
+    background-color: olivedrab;
+    color: white;
+    width: 100%;
+    font-size: 0.75em;
+    font-weight: bold;
+    height: 1.5em;
+    border: none;
+    border-radius: 4px;
+    margin-bottom: 1em;
+}
+.shopping-grid-squares .shopping-grid-squares-text button {
+    background-color: olivedrab;
+    color: white;
+    width: 100%;
+    font-size: 1em;
+    font-weight: bold;
+    height: 2em;
+    border: none;
+    border-radius: 8px;
+    margin-bottom: 1em;
+}
+.shopping-grid-squares .shopping-grid-squares-text button:hover {
+    background-color: darkgreen;
+    color: lightgray;
+    width: 100%;
+    font-size: 1em;
+    font-weight: bold;
+    height: 2em;
+    border: none;
+    border-radius: 8px;
+    margin-bottom: 1em;
+}
+#input-index {
+    background-color: lightgrey;
+    color: navy;
+    padding: 0.5em 2em;
+    font-size: 1.5em;
+    font-weight: bold;
+    height: 3em;
+    border: none;
+    border-radius: 8px;
+}
+#index-form {
+    width: 25%;
+    padding: 4em 0em 8em;
+    align-content: center;
+    transform: translate(162%, 0%);
+}
+.nav-bar {
+    width: 100%;
+    background-color: lightgray;
+    height: 4em;
+    font-size: 2em;
+    position: relative;
+    text-align: center;
+}
+.nav-bar-links{
+    color: navy;
+    position: absolute;
+    top: 50%;
+    left: 20%;
+    transform: translate(-50%, -50%);

```

Comment:
You can handle the positioning without using position or the transform. In fact for this app you shouldn't need to use transform at all. I'd really recommend taking all of them out.